File: app/src/main/java/com/escodro/music/rest/spotify/model/Image.java
package com.escodro.music.rest.spotify.model;

import com.escodro.music.rest.spotify.SpotifyAPI;

/**
 * Object to represent an {@link SpotifyAPI} {@link Image}.
 */
public class Image {
    /**
     * {@link Image} height.
     */
    private String height;

    /**
     * {@link Image} width.
     */
    private String width;

    /**
     * {@link Image} url.
     */
    private String url;

    public String getHeight() {
        return height;
    }

    public void setHeight(String height) {
        this.height = height;
    }

    public String getWidth() {
        return width;
    }

    public void setWidth(String width) {
        this.width = width;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }
}

File: src/main/kotlin/com/escodro/savel/data/remote/spotify/model/artist/Image.kt
package com.escodro.savel.data.remote.spotify.model.artist

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Image(
    @SerialName("url")
    val url: String,
    @SerialName("height")
    val height: Int,
    @SerialName("width")
    val width: Int,
)
